webpack.config.common.js

```diff
@@ -75,7 +75,6 @@ const config = {
           {
             loader: "sass-loader",
             options: {
-              minimize: true,
```

Comment:
What happened here? Is it not minimizing?

Reply:
Sass-loader 8 doesn't do minimizing, so it's not a valid option. It wasn't doing anything before I believe; the `cssnano` plugin for `postcss-loader` does the bulk of the work.

Reply:
ok, interesting

Reply:
Yeah, the css is the same. `dist/main.bundle.min.js` is 50 kb bigger.